src/main/java/frc/robot/Constants.java

```diff
@@ -104,8 +104,11 @@ public static final class Climber {
     }
 
     public static final class AmpBar {
-        public static final double IN = 0.45;
+        public static final double IN = 0;
         public static final double OUT = IN + 0.2506;
```

Comment:
fyi: this value is likely too small. for reference, in -> out on the intake is 37.